.github/workflows/contrib-tests.yml

```diff
@@ -0,0 +1,28 @@
+name: Contrib tests
+
+on: workflow_dispatch
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      # Install huggingface_hub
+      - name: Install `huggingface_hub`
+        run: |
+          pip install --upgrade pip
+          pip install .
+
+      # Install downstream libraries
+      - name: Install downstream libraries
+        run: pip install -r contrib/requirements.txt
```

Comment:
For multiple versions instead of inputs we could maybe use a github actions matrix to go through different ones?

Reply:
You're right, will be a nightmare to handle. Let's have separate venvs (see https://github.com/huggingface/huggingface_hub/pull/1200#issuecomment-1322444901)